VIBE.md

```
@@ -0,0 +1,474 @@
+
+Let's create a copy of the `/truth` mount so that `claude` can make all the
+necessary changes on its copy, not the host version of `changelog.com`:
+
+```fish
+cp -r /truth ~/workspace
+```
+
+Now fire up `claude` - there is a `c<SPACE>` shell abbreviation that has
+all the right details in place:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)